app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,16 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    user = User.find_by email: params[:email]
+    if user && !user.activated? && user.authenticated?(:activation, params[:id])
+      user.activate
+      user.update_attribute :activated, true
+      user.update_attribute :activated_at, Time.zone.now
```

Comment:
In activate action, you wrote updating for activated and activated_at columns, therefore you don't need to write line 6 & 7 here 
```suggestion
```